src/main/kotlin/com/sourcegraph/cody/chat/actions/BaseCommandAction.kt

```diff
@@ -12,12 +11,9 @@ abstract class BaseCommandAction : BaseChatAction() {
   abstract val myCommandId: CommandId
 
   override fun doAction(project: Project) {
-
     FileEditorManager.getInstance(project).selectedTextEditor?.let {
-      CodyEditorFactoryListener.Util.informAgentAboutEditorChange(it, hasFileChanged = false) {
```

Comment:
Because we do it when file changes focus, is opened, closed, etc.
So that info is already in the agent.
Previously it was not working because in some cases we were failing to properly propagate info albo cursor and selection.

Reply:
got it, thanks!